### PR TITLE
Scenarios release

### PR DESCRIPTION
**Pull request by Argn0**

### CODE REVIEW

**Comment by howardchung** [comment]
@Argn0 do you want to wait for the next release?  (probably in a week or so)

**Comment by Argn0** [comment]
@howardchung ok

**Comment by bdbch** [comment]
Could be merged now, right?

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ class Header extends React.Component {
       <Link key="header_heroes" to="/heroes">{strings.header_heroes}</Link>,
       <Link key="header_distributions" to="/distributions">{strings.header_distributions}</Link>,
       <Link key="header_records" to="/records">{strings.header_records}</Link>,
-      // <Link key={strings.header_scenarios} to="/scenarios/itemTimings">{strings.header_scenarios}</Link>,
+      <Link key={strings.header_scenarios} to="/scenarios/itemTimings">{strings.header_scenarios}</Link>,
```
Comment (howardchung) [inline src/components/Header/Header.jsx]: should this just be /scenarios?  Also update the key to be a string (it doesn't need  to be localized)

Comment (howardchung) [reply src/components/Header/Header.jsx]: yeah but i think the link should be just to /scenarios so if you change the default later you don't have to update this

**Comment by Argn0** [comment]
fixed default URL

**Pull request merged**